internal/blockchain/ethereum/abi_definitions.go

```diff
@@ -96,3 +96,16 @@ var batchPinEventABI = ABIElementMarshaling{
 		},
 	},
 }
+
+var networkVersionMethodABI = ABIElementMarshaling{
+	Name:            "networkVersion",
+	Type:            "function",
+	StateMutability: "pure",
+	Inputs:          []ABIArgumentMarshaling{},
+	Outputs: []ABIArgumentMarshaling{
+		{
+			InternalType: "uint8",
```

Comment:
I wonder whether a `bytes32` would be the most efficient option here, and using a `semver` structure. `uint8` gets stored as a `unit256` anyway.

Reply:
(added more discussion on this in a separate comment)